api.go

```diff
@@ -61,3 +62,18 @@ func (es *ExploreServer) apiGetBlockData(start types.BlockHeight, end types.Bloc
 
 	return blockSummaries, err
 }
+
+// apiGetHash queries the running instance of siad and returns the raw
+// data to the calling function. The json data could decode into
+// multiple types of structures, all of which *should* have the
+// ResponseType field as a string for identifying the type of data
```

Comment:
I'd do the above docstring like this:

`apiGetHash returns the json of the object represented by the hash. Multiple object types are supported. The object type can be determined by the `ResponseType` field in the json`.